<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Photo broom</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/gui/main-ico.svg</normaloff>:/gui/main-ico.svg</iconset>
  </property>
  <widget class="QStackedWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCollection">
    <property name="title">
     <string>Photo collection</string>
    </property>
    <addaction name="actionNew_collection"/>
    <addaction name="actionOpen_collection"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuPhotos">
    <property name="title">
     <string>Photos</string>
    </property>
    <addaction name="actionAdd_photos"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuWindows">
    <property name="title">
     <string>Windows</string>
    </property>
   </widget>
   <addaction name="menuCollection"/>
   <addaction name="menuPhotos"/>
   <addaction name="menuWindows"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="rightDockWidget">
   <property name="windowTitle">
    <string>Photo information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="TagEditorWidget" name="tagEditor"/>
  </widget>
  <action name="actionNew_collection">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_collection">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionAdd_photos">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add photos</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About Photo Broom</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TagEditorWidget</class>
   <extends>QWidget</extends>
   <header>widgets/tag_editor/tag_editor_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
